src/assets/scss/components/_signInForm.scss

```diff
@@ -57,11 +57,13 @@ sign-in-form {
     font-weight: bold;
     gap: 0.5rem;
     padding-block: 20px;
+    transition: cubic-bezier(1, 0, 0, 1) 0.3s;
 
     &:hover {
       border-color: $okta-brand;
       background-color: $okta-brand;
       color: $colorCru-white;
+      transition: cubic-bezier(1, 0, 0, 1) 0.3s;
```

Comment:
This is a duplication. You don't need to define `transition` on hover as well. the only reason why you would is to make it have a different effect on hover-off.